delete-user-data/CHANGELOG.md

```diff
@@ -2,3 +2,7 @@
 
 feature - Add a new param for recursively deleting subcollections in Cloud Firestore (issue #14).
 fixed - Fixed "cold start" errors experienced when the extension runs after a period of inactivity (issue #48).
+
+## Version 0.1.0
```

Comment:
the initial release of this extension was actually 0.1.1, we skipped 0.1.0 due to an issue.

Reply:
Updated